### DISCUSSION TITLE
Unable to register for Github Student benefits due to Github requiring incorrect school email.

### DISCUSSION THREAD

**Discussion by onjahhh** [community/community#42709] [GitHub Education]
I am currently trying to register for Github Student benefits but at the first step of the registration process, Github requires a specific email address that my school, to my knowledge, no longer uses. I cannot find any way to get around this step without changing the school.
![schoolemail](https://user-images.githubusercontent.com/73459195/209674586-85bf86b4-3a45-4d12-8b0d-b1ae24cf6f5e.png)
![schoolrequiredemail](https://user-images.githubusercontent.com/73459195/209674655-4160fa00-1189-4aa1-85d1-0d652d2a0017.PNG)

**Comment by hiten36** [comment]
Try to submit it without school email if possible. Or you can simply reset and submit again. If nothing helps you can always mail github. 

There are some discussions on this topic that have been resolved. Here are some links that may help you.

https://github.com/orgs/community/discussions/38918
https://github.com/orgs/community/discussions/30138
https://github.com/orgs/community/discussions/37159
